scio-grpc/src/main/java/com/spotify/scio/grpc/GrpcBatchDoFn.java

```diff
@@ -128,7 +128,7 @@ public static class Builder<
     private SerializableFunction<List<Input>, BatchRequest> batchRequestFn;
     private SerializableFunction<BatchResponse, List<Pair<String, Output>>> batchResponseFn;
     private SerializableFunction<Input, String> idExtractorFn;
-    private Integer maxPendingRequests;
+    private int maxPendingRequests = GrpcDoFn.DEFAULT_MAX_PENDING_REQUESTS;
     private Integer batchSize;
```

Comment:
We have a default for `maxPendingRequests` in the non batched.
Shoulw we also add a default for the batch size or require the user to provide one ?

Reply:
IMHO, when knowing the target system, I'm not super keen on setting defaults. Even for the max pending requests. Just added the value for consistency.

Reply:
Ohh .. i completely missunderstood your question !! 

I thought we needed a default for the `batchSize` and that was my answer to that !! 

but of course if we have a default in one we should totally have in the other. That makes perfect sense !